Xamarin.Forms.Controls/GalleryPages/DynamicViewGallery.cs

```diff
@@ -106,15 +107,22 @@ void OnElementUpdated(View oldElement)
 
 			var elementType = _element.GetType();
 
+			GetProperties(_element, elementType, _propertyLayout);
```

Comment:
`elementType` can be removed from the passed variables, because is just `_element.GetType()`;